scripts/kani-fmt.sh

```diff
@@ -32,7 +32,7 @@ for suite in "${TESTS[@]}"; do
     set -f; IFS=$'\n'
     files=($(find "${suite}" -name "*.rs"))
     set +f; unset IFS
-    rustfmt --unstable-features "$@" "${files[@]}" || error=1
+    rustfmt --unstable-features "$@" "${files[@]}" --edition 2021 || error=1
```

Comment:
Can this be added to `rustfmt.toml` instead? If possible, I'd rather put it there instead of passing as a parameter in  this script.

Reply:
Good point! Done